mylist.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
```

Comment:
- [ ] Hi as required and as mentioned by the previous reviewer, the `each` method should yield all members of @list in succession, but you shouldn't use the ruby `each` keyword method. you have to design it yourself